packages/fxa-settings/src/components/PageSettings/index.stories.tsx

```diff
@@ -8,6 +8,13 @@ import { PageSettings } from '.';
 import { LocationProvider } from '@reach/router';
 import AppLayout from '../AppLayout';
 import { MockedCache, mockEmail } from 'fxa-settings/src/models/_mocks';
+import { MOCK_SERVICES } from '../ConnectedServices/MOCK_SERVICES';
+
+const SERVICES_NON_MOBILE = MOCK_SERVICES.filter(el => {
+  if (el.deviceType === 'mobile') return false;
+  if (el.name.toLowerCase().includes('ipad')) return false;
+  return true;
```

Comment:
I vaguely remember seeing this logic somewhere else.  It would be nice to capture it in one place.

Reply:
haha I'm scrolling _up_... sorry I didn't mean somewhere else _in this PR_.  I meant before this PR.

Reply:
> 
> 
> I don't know of any other place we filter like this, there is a place where we make [similar checks to show 
the icons](https://github.com/mozilla/fxa/pull/6677),

Hmm maybe it was in another package where I saw that logic.  So for this, looks like we need something that tells us if a device is a "mobile" device or not.  i.e.

```JavaScript
const isMobileDevice = (d) => d.deviceType === 'mobile' || d.name.toLowerCase().includes('ipad');
```

For the first filter we can just use `.filter(isMobileDevice)`.  For the second one, we do need to do `.filter(x => !isMobileDevice(x)`.

In addition having the "mobile device" logic in one place, I think it also reads better when we give that logic a name.

> but that would be weird to pull out into this story IMO

I don't understand what's weird about it.  I saw it in this PR so I suggested here.  If there's a more appropriate story, then let me know and I'll leave a note there.